### PR TITLE
add config values for GCS deep storage

### PR DESCRIPTION
**Pull request by Caroline1000**

### CODE REVIEW

File: docs/content/development/extensions-contrib/google.md [hunk 1]
```diff
@@ -8,12 +8,15 @@ To use this extension, make sure to [include](../../operations/including-extensi
 
 ## Deep Storage
 
-[Google Cloud Storage](https://cloud.google.com/storage/) is another option for deep storage. This requires some additional druid configuration.
+Deep storage can be written to Google Cloud Storage either via this extension or the [druid-hdfs-storage extension](../extensions-core/hdfs.html).
+
+### Configuration
+
+|Property|Possible Values|Description|Default|
```
Comment (jihoonson) [inline docs/content/development/extensions-contrib/google.md]: I think we already have a document for this. Please check http://druid.io/docs/latest/development/extensions-core/hdfs.html#google-cloud-storage.

The removed configurations are actually valid when directly using GCS without Hadoop.

Comment (Caroline1000) [reply docs/content/development/extensions-contrib/google.md]: We have two extensions that enable using GCS in deep storage: ../extensions-contrib/google.md and ../extensions-core/hdfs.html. When using the google extension, the `druid.storage.type` needs to be `google`, which isn't documented anywhere as far as I know. I was hoping to make it more clear how to use GCS with either this extension or the hdfs extension.

Comment (jihoonson) [reply docs/content/development/extensions-contrib/google.md]: Thanks, I got your point. 

`druid.storage.type` should be specified as you said. But, `druid.storage.storageDirectory` is used only in HDFS or local deep storage types. GCS deep storage type needs to set `druid.google.bucket` and `druid.google.prefix` instead.

Comment (Caroline1000) [reply docs/content/development/extensions-contrib/google.md]: Ah okay. Thanks, Jihoon!

**Comment by stevenchen3** [comment]
It needs the following environment variable for authentication as well. It bugs me a bit at the beginning. We might need to clearly document it as well.

```
export GOOGLE_APPLICATION_CREDENTIALS=/path/to/keyfile
```

**Comment by jihoonson** [comment]
@stevenchen3 thanks. Are you interested in making a PR for that?

**Pull request merged**